ambry-api/src/main/java/com.github.ambry/network/ServerNetworkResponseMetrics.java

```diff
@@ -25,16 +25,18 @@
   private final Histogram responseTotalTime;
   private final Histogram responseSendTimeBySize;
   private final Histogram responseTotalTimeBySize;
+  private final Histogram replicationResponseTotalTime;
   private long timeSpentTillNow;
 
   public ServerNetworkResponseMetrics(Histogram responseQueueTime, Histogram responseSendTime,
       Histogram responseTotalTime, Histogram responseSendTimeBySize, Histogram responseTotalTimeBySize,
-      long timeSpentTillNow) {
+      Histogram replicationResponseTotalTime, long timeSpentTillNow) {
```

Comment:
I don't think we should introduce another histogram. Instead we should send in different ones for normal get v/s replication get

Reply:
This needs to change the logic outside this method. Let me try.